webapp/components/file_attachment.jsx

```diff
@@ -120,7 +120,37 @@ class FileAttachment extends React.Component {
             var re3 = new RegExp('\\)', 'g');
             var url = fileUrl.replace(re1, '%20').replace(re2, '%28').replace(re3, '%29');
 
-            $(imgDiv).css('background-image', 'url(' + url + '_thumb.jpg)');
+            $(imgDiv).html("<img src='" + url + "_preview.jpg'/>");
```

Comment:
we should be moving away from jquery, but since it was there before I'm Ok with it (just a friendly reminder)